go-fuzz/main.go

```diff
@@ -57,6 +58,16 @@ func main() {
 		log.Fatalf("both -http and -worker are specified")
 	}
 
+	if *flagDict != "" {
+		dictStat, err := os.Stat(*flagDict)
```

Comment:
Add a comment explaining why we stat here.

Reply:
Could we pass *flagDict to workerMain? Or read here and pass contents to workerMain?
It looks like this split stat/size check + reading separately doubles amount of code.